pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
I'm not seeing it.  I just see a static list being copied back: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/client/fake_namespaces.go#L33, that's better than the empty ones in origin?